sshsysmon/templates/htmldash.hb

```diff
@@ -80,7 +80,7 @@ Colors from: http://clrs.cc/
 			<div class="container-fluid">
 				<div class="row">
 					{{!-- Alarms --}}
-					<div class="col-xs-4 inspector">
+					<div class="col-xs-12 col-md-4 inspector">
```

Comment:
What's this change for?

Reply:
This was some formatting fix. The columns started overlapping on mobile devices and this gives each chart the full width if a tiny display is used.